### PR TITLE
fix for search dropdown on small screen + icon overlapping fix

### PR DESCRIPTION
**Pull request by Ijin08**

### CODE REVIEW

**Comment by torkelo** [comment]
This does not work for me, uses mobile rule for every screen size (no sidebar or search filterbox)

**Comment by daniellee** [comment]
@torkelo strange, works on both Chrome and Firefox on desktop and safari/firefox on iPhone for me.

image

and 

image

**Comment by torkelo** [comment]
well that is because Patrick fixed it after my comment :)

File: public/sass/components/_search.scss [hunk 1]
```diff
@@ -230,3 +230,24 @@
...
+    display: none;
+  }
+
+  .search-item__tags {
+    display: none;
+  }
+
+  input[type='text'],
+  input[type='number'],
+  textarea {
```
[the remark this answers is not in the document]
Comment (Ijin08) [reply public/sass/components/_search.scss]: Missed this, will look into it now.

**Pull request merged**